pkg/Makefile

```diff
@@ -8,12 +8,13 @@ CONFIG_DIR = /etc/aerospike
 
 # Package variables
 NAME = "asconfig"
-VERSION = $(shell git describe --tags --always | cut -c 2-)
```

Comment:
Could you make sure to test installing this rpm? I remember at one time, there was an issue specifically with rpm packages not liking certain characters. Specifically '-'.

http://ftp.rpm.org/max-rpm/ch-rpm-file-format.html

Reply:
You're right, fpm auto converts them to `_` for rpms so this is ok. The RPM install was failing because the package was missing etc/aerospike. It's fixed now